libs/components/forms/src/lib/modules/file-attachment/types/file-attachment-change.ts

```diff
@@ -2,7 +2,7 @@ import { SkyFileItem } from '../file-item';
 
 export interface SkyFileAttachmentChange {
   /**
-   * The file that was added or removed.
+   * The file that was added. If the file was removed this property will be `undefined`.
```

Comment:
The first sentence reads a little weird with the new sentence. How about “The file that was added, or `undefined` if the file was removed.”